addons/packages/kpack/0.5.3/README.md

```diff
@@ -0,0 +1,673 @@
+# kpack
+
+[kpack](https://github.com/pivotal/kpack) utilizes unprivileged Kubernetes
+primitives to provide builds of OCI images as a platform implementation
+of [Cloud Native Buildpacks](https://buildpacks.io) (CNB).
+
+## Components
+
+* kpack
+
+## Supported Providers
+
+The following table shows the providers this package can work with.
+
+| AWS | Azure | vSphere | Docker |
+|-----|-------|---------|--------|
+| ✅   | ✅     | ✅       | ✅      |
+
+## Configuration
+
+The following configuration values can be set to customize the kpack
+installation.
+
+### kpack Configuration
+
+| Value                            | Required/Optional | Description                                                                                                                                                                                                                                               |
+|----------------------------------|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `kp_default_repository`          | Required          | Docker repository used for builder images and dependencies. Ex:<br>Dockerhub: `mydockerhubusername/my-repo`<br>GCR: `gcr.io/my-project/my-repo`<br>Harbor: `myharbor.io/my-project/my-repo`. Required by the [kp cli](https://github.com/vmware-tanzu/kpack-cli).|
```

Comment:
@tylerphelan This line is failing our Markdown linter because of the `<br>`'s. I know that they are there to format the contents of the cell better. ATM I don't have a better suggestion other than to remove them.

Reply:
thanks!